Dockerfile

```
@@ -1,149 +1,207 @@
-    fi
-
-# Clone and install Double-Down
-RUN if [ "$include_dagmc" = "true" ] ; \
-    then git clone --single-branch --branch main https://github.com/pshriwise/double-down.git ; \
-    cd double-down ; \
-    mkdir build ; \
-    cd build ; \
-    cmake .. -DCMAKE_INSTALL_PREFIX=.. \
-             -DMOAB_DIR=/usr/local \
-             -DEMBREE_DIR=/embree ; \
-    make -j"$compile_cores" ; \
-    make -j"$compile_cores" install ; \
-    fi
-
-# Clone and install DAGMC
-RUN if [ "$include_dagmc" = "true" ] ; \
-    then mkdir DAGMC ; \
-    cd DAGMC ; \
-    git clone --single-branch --branch 3.2.0 https://github.com/svalinn/DAGMC.git ; \
-    mkdir build ; \
-    cd build ; \
-    cmake ../DAGMC -DBUILD_TALLY=ON \
-                   -DCMAKE_INSTALL_PREFIX=/DAGMC/ \
-                   -DMOAB_DIR=/usr/local \
-                   -DBUILD_STATIC_LIBS=OFF \
-                   -DBUILD_STATIC_EXE=OFF ; \
-    make -j"$compile_cores" install ; \
-    rm -rf /DAGMC/DAGMC /DAGMC/build ; \
-    fi
-
-# Clone and install OpenMC with DAGMC
-RUN if [ "$include_dagmc" = "true" ] ; \
-    then git clone --recurse-submodules https://github.com/openmc-dev/openmc.git /opt/openmc ; \
-    cd /opt/openmc ; \
-    mkdir build ; \
-    cd build ; \
-    cmake -Doptimize=on \
-          -Ddagmc=ON \
-          -DDAGMC_DIR=/DAGMC/ \
-          -DHDF5_PREFER_PARALLEL=on ..  ; \
-    make -j"$compile_cores" ; \
-    make -j"$compile_cores" install ; \
-    cd ..  ; \
-    pip install -e .[test] ; \
-    fi
+# clone and install openmc
+RUN mkdir -p ${HOME}/OpenMC && cd ${HOME}/OpenMC \
+    && git clone --shallow-submodules --recurse-submodules -b ${openmc_branch} --depth=1 ${OPENMC_REPO} \
+    && mkdir build && cd build ; \
+    if [ ${build_dagmc} = "on" ] && [ ${build_libmesh} = "on" ]; then \
+        cmake ../openmc \
+            -Doptimize=on \
+            -Ddebug=on \
+            -DHDF5_PREFER_PARALLEL=on \
+            -Ddagmc=on \
+            -Dlibmesh=on \
+            -DCMAKE_PREFIX_PATH="${DAGMC_INSTALL_DIR};${LIBMESH_INSTALL_DIR}" ; \
+    fi ; \
+    if [ ${build_dagmc} = "on" ] && [ ${build_libmesh} = "off" ]; then \
+        cmake ../openmc \
+            -Doptimize=on \
+            -Ddebug=on \
+            -DHDF5_PREFER_PARALLEL=on \
+            -Ddagmc=ON \
+            -DCMAKE_PREFIX_PATH=${DAGMC_INSTALL_DIR} ; \
+    fi ; \
+    if [ ${build_dagmc} = "off" ] && [ ${build_libmesh} = "on" ]; then \
+        cmake ../openmc \
+            -Doptimize=on \
+            -Ddebug=on \
+            -DHDF5_PREFER_PARALLEL=on \
+            -Dlibmesh=on \
+            -DCMAKE_PREFIX_PATH=${LIBMESH_INSTALL_DIR} ; \
+    fi ; \
+    if [ ${build_dagmc} = "off" ] && [ ${build_libmesh} = "off" ]; then \
+        cmake ../openmc \
+            -Doptimize=on \
+            -Ddebug=on \
+            -DHDF5_PREFER_PARALLEL=on ; \
+    fi ; \
+    make 2>/dev/null -j${compile_cores} install \
+    && cd ../openmc && pip install -e .[test, depletion-mpi]
```

Oops. It seems that there cannot be a space after the comma here. My fault.

```suggestion
    && cd ../openmc && pip install -e .[test,depletion-mpi]
```

---

I didn't know that rule, pep8 is keen on spaces after commas in a list so I would have guessed it was fine for pip commands as well

---

Me either and I had the same thought about pep8. Might make command line argument parsing easier I suppose.